ch_pipeline/hfb/io.py

```diff
@@ -0,0 +1,99 @@
+"""HFB-related tasks
```

Comment:
This module does not contain tasks.

Reply:
Moved `HFBReader` back into the `hfb.containers` module and removed `hfb.io` for now (the way you originally set it up in `core.containers`). A future PR (#139) adds `hfb.io` again, putting `HFBDataLoader` in there. That `HFBDataLoader` is a task, so we can make the module docstring, for example,
```
"""HFB tasks related to reader and writing data
"""
```